.github/workflows/test-linux.yml

```diff
@@ -12,8 +18,14 @@ on:
       - canary
       - v[0-9]+.x-staging
       - v[0-9]+.x
-    paths-ignore:
-      - "README.md"
+    paths:
+      - '*'
+      - !'.mailmap'
+      - !'**.md'
```

Comment:
Don't we need to keep this for doc changes? Or at least for changes in `errors.md`, as IIRC we have a test to check that all errors are documents.
```suggestion
```